Jenkinsfile

```diff
@@ -1 +1,11 @@
-buildPlugin(findbugs: [run: true, archive:true, unstableTotalAll: "0"])
+buildPlugin(
+    findbugs: [run: true, archive:true, unstableTotalAll: "0"],
+    configurations: [
+        [ platform: "linux", jdk: "8", jenkins: null ],
+        [ platform: "windows", jdk: "8", jenkins: null ],
+        [ platform: "linux", jdk: "8", jenkins: "2.150.1", javaLevel: "8" ],
+        [ platform: "windows", jdk: "8", jenkins: "2.150.1", javaLevel: "8" ],
```

Comment:
:question: The addition below for JDK11 does make sense. 

But the two lines above adding `jenkins.version=2.150.1` seem unrelated. I mean, why not, but it _looks_ slightly out of scope here.

Reply:
anyway, I think this is not working as the build only ran on linux/windows 8, nothing in 11.

Reply:
I've changed the Jenkinsfile after the merge so that it only runs the jdk 11 configurations if anyone of you guys are the PR author.
Too many test failures :)